tests/network/primary_pod_network.go

```diff
@@ -98,12 +98,12 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 
 				It("should report PodIP/s IPv4 as its own on interface status", func() {
 					vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
-					Eventually(vmiIP).Should(Equal(vmiPod.Status.PodIP), "should contain VMI Status IP as Pod status ip")
-					Eventually(vmiIPs).Should(ContainElement(vmiPod.Status.PodIP), "should contain IPv4 reported by guest agent")
+					Eventually(vmiIP, 30*time.Second, 2*time.Minute).Should(Equal(vmiPod.Status.PodIP), "should contain VMI Status IP as Pod status ip")
```

Comment:
Eventually(ACTUAL, (TIMEOUT), (POLLING_INTERVAL), (context.Context)).Should(MATCHER)

i think the order is vice versa?
moreoever no need to poll every 30, better poll every 5 seconds please
it will make the tests faster

Reply:
Fixed